doc/getting_started/index.rst

```diff
@@ -29,85 +29,53 @@ Follow one of the following guides for your host operating system.
 Get the source code
 *******************
 
-Zephyr's multi-purpose :ref:`west` tool lets you easily get the Zephyr project
-source code, instead of manually cloning the Zephyr repos along with west
-itself.
-
-.. warning::
-
-   It's possible to use Zephyr without installing west, but you have
-   to **really** know :ref:`what you are doing <no-west>`.
+Zephyr's multi-purpose :ref:`west <west>` tool lets you easily get the Zephyr
+project repositories. (It's possible to use Zephyr without installing west, but
```

Comment:
Hmm... that's longer, not shorter (the main goal of this PR is to make it shorter and easier to read), and the sentence I think you find unkind ("It's possible ... really know what you're doing") is already part of this document.

I personally don't take that sentence as unkind (though I wrote it originally, so grain of salt there). I also disagree with the suggestion; experts are experts and I certainly wouldn't make that recommendation to one.

To try to preserve the meaning without making it longer, I'll switch to:

```
(It's possible to use Zephyr without installing west, but
:ref:`that's documented elsewhere <no-west>`.)
```

Reply:
How about "While it's easiest to develop with Zephyr using west, we also show the ..."?

West makes a lot of things much more convenient, but there is a small, vocal, and considered minority with valid use cases to not use it. So for me personally authoring this PR and showing up in the `git blame` lines, "recommend" is not the word I would use. Feel free to submit your own changes making recommendations, of course :).

Reply:
> I think the point was to say using west is the recommended approach.

Changed to this:

```
While it's easiest to develop with Zephyr using west,
we also have :ref:`documentation for doing without it <no-west>`.
``

Reply:
Either "it's easiest" or "we recommend" is fine, they mean pretty much the same thing unless you're the person who has been receiving a lot "constructive" feedback from a vocal minority of people trying to avoid west.

"Make simple things [west] simple, and complex things [no west] possible"

Don't let people complaining that "the complex is not easy" affect you. Just keep it "possible" that's all :-)

PS: this thread is not available in the "Files Changed" tab any more. I think it's (again) because of force-pushes.

Reply:
As Rich Hickey eloquently argued in Simple Made Easy (https://www.youtube.com/watch?v=34_L7t7fD_U), easiest isn't always best :smile: